src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -958,22 +1092,60 @@ public void parse(InputFile inputFile, Format ext, int type, boolean thumbOnly,
 
 						if (sz > 0 && !net.pms.PMS.isHeadless()) {
 							BufferedImage image = ImageIO.read(new ByteArrayInputStream(thumb));
```

Comment:
I've removed ```!net.pms.PMS.isHeadless()``` from this if, built and run it on a headless Centos server. Thumbnails are generated just fine, and the overlay work as well. Am I doing something wrong when testing? Do I have to do something special for this code to be used? If not, it seems like we can just remove that check and thumbnails will work for headless installations as well.